### ISSUE TITLE
tldr page for command docker---markdown not found

### ISSUE THREAD

**Issue by qshzh** [denisidoro/navi#497]
**Describe the bug**
```shell
> navi --tldr docker

Failed to call:
tldr docker --markdown

Output:
tldr page for command docker---markdown not found


Error:


Note:
Please make sure you're using a version that supports the --markdown flag.
If you are already using a supported version you can ignore this message.
The tldr client written in C (the default one in Homebrew) doesn't support markdown files, so navi can't use it.
The client written in Rust is recommended. The one available in npm works, too.

Failed to read /dev/tty
```

**To Reproduce**
1. Download the pre-compiled binaries
2. install denisidoro/cheats
3. run the command

**Expected behavior**

**Screenshots**

**Versions:**
 - OS:  ubuntu 20.04
 - Shell Version zsh 5.8

**Additional context**

Thanks for your amazing tool and contribution.

**Comment by denisidoro** [comment]
Could you please try installing a C client for TLDR? Or the JS one.

This should do the trick:
```bash
npm install -g tldr
```

**Comment by qshzh** [comment]
Thanks for your help. I installed the C client and did not work. JS one works.
